src/Sarif.PatternMatcher.Cli/Sarif.PatternMatcher.Cli.csproj

```diff
@@ -21,7 +21,6 @@
     <PackageReference Include="Microsoft.Data.Sqlite" Version="5.0.5" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
-    <PackageReference Include="xunit" Version="2.4.2" />
```

Comment:
this one took me sometime to find, with this, dotnet test has different behavior in Windows and Linux,
in Linux it try to run this project as test project.